### DISCUSSION TITLE
Install help for n00b

### DISCUSSION THREAD

**Discussion by bertnur** [FossifyOrg/General-Discussion#59] [Q&A] [answered]
Hi,
I'm a standard end user with limited Gitlab experience. Is there a simple way to install the fossify apps on an Android phone directly? I have installed lots from apk in the past, but I don't see any apk for these apps.

**Comment by Aga-C** [comment, marked as answer]
Apps aren't released yet. Please wait few more days and APKs will be available to download from GitHub.

**Reply by bertnur** [reply]
Great-thanks
